src/Design.cc

```diff
@@ -98,6 +98,249 @@ void Design::readDef(const std::string& file_name,
                child);
 }
 
+
+
+odb::dbITerm* Design::staToDBPin(const sta::Pin *pin) {
+  ord::OpenRoad *openroad = ord::OpenRoad::openRoad();
+  sta::dbNetwork *db_network = openroad->getDbNetwork();
+  odb::dbITerm *iterm;
+  odb::dbBTerm *bterm;
+  db_network->staToDb(pin, iterm, bterm);
+  return iterm;
+}
+
+sta::PinSet Design::findStartPoints() {
+  sta::Network *network = sta::Sta::sta()->cmdNetwork();;
+  sta::PinSet pins(network);
+  sta::VertexPinCollector visitor(pins);
+  sta::Sta::sta()->visitStartpoints(&visitor);
+  return pins;
+}
+
+sta::PinSet Design::findEndpoints() {
+  sta::Network *network = sta::Sta::sta()->cmdNetwork();;
+  sta::PinSet pins(network);
+  sta::VertexPinCollector visitor(pins);
+  sta::Sta::sta()->visitEndpoints(&visitor);
+  return pins;
+}
+
+std::vector<std::string> Design::extractEndpointNames() {
```

Comment:
Why does this need to be a method?  It appears to be written in terms of public methods